day6/salestax.py

```diff
@@ -0,0 +1,37 @@
+import csv
+import pandas
+import fs as fs
```

Comment:
why `import fs as fs`? you don't need `as fs` here as it is already imported as `fs`

Reply:
okay